README.md

```diff
@@ -1 +1,28 @@
+
+Hello! My name is David Mendieta, and I am from Los Angeles, California. I am currently a third-year student majoring in Public Policy, and I aim to attend law school in the near future. While I am sad that I am away from family and friends at the moment, I am very excited to take Computing for the Social Sciences and see how I can better improve my skills in coding and R!
+
+### L.A. to Chicago
+
+![Image of Santos L Halper](https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/67050843_2361407843913794_3635612463702999040_n.jpg?_nc_cat=107&_nc_sid=7aed08&_nc_ohc=NP7gpxVZKbkAX_QtzOb&_nc_ht=scontent-ort2-1.xx&oh=009eb9e1b6b86cc2aa267f6cab6ca404&oe=5EB9C811)
+
+I originally grew up in the neighborhood of _Koreatown_, a 15 minute drive from Downtown Los Angeles. I always lived with 6-8 people in my small, two-bedroom apartment, where I shared a space with my parents, older sister and brother, aunt, uncle, and two cousins - occasionally. What we lacked in physical space we made up with love and a strong support system, as we did whatever we could to make ends meet and have as stable of a life we could afford. My siblings had already been to college, but they stayed within the LA area. Hoping to branch out and see another part of the United States, I applied and recieved admission to UChicago as a **Quest Bridge Scholar**. I was nervous about leaving my family to attend school in a city I've never visited, but throughout the years my close friends have made it so easy every step of the way.
+
+### What Do I Do Now
+
+Right now, I am a Policy Intern for the _**Chicago Coalition for the Homeless**_. This academic year-long commitment involves compiling data for the organization's yearly [Chicago Homeless Estimate](https://www.chicagohomeless.org/faq-studies/), calculating how many Chicago residents experience homelessness in a given year. My primary tasks involve co-writing the report, detailing the methodology behind the data analysis, and recently I assisted my supervisor in gathering some first-hand accounts from people who once experienced homelessness in Chicago. When I am not working for the coalition, I am involved in a number of activities:
+* Dancing for the OLAS Cultural Show each year
+* Working in the Reynolds Basement for Financial Operations - _the best job ever!_
+* Cooking Chicken Noodle Soup each Tuesday for my roommate and one of my best friends as he comes home from tutoring
+* Sleeping as much as I possibly can in one day
+
+![Image of my friends and I at the OLAS Cultural Show](https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/35488753_1764654176922500_3912556108182454272_n.jpg?_nc_cat=102&_nc_sid=8024bb&_nc_ohc=psO1xIHI_8YAX-mOWT1&_nc_ht=scontent-ort2-1.xx&oh=8f53fb8b9831797d977f293304971513&oe=5EB8F69B)
+
+### The End
+
+That's just a little bit about me, and I hope that this brief introduction shed some light on myself and the things I find most interesting! Thank you!
```

Comment:
Good job! You seem to have incorporated all the required elements for the assignment.